### PR TITLE
feat(platforms): Add SvelteKit as a platform (frontend)

### PR DESCRIPTION
**Pull request by Lms24**
This PR adds SvelteKit as a platform to the Sentry frontend since the SvelteKit SDK is nearing GA. 

Going to merge this only after https://github.com/getsentry/sentry-docs/issues/6843 is done.

### CODE REVIEW

**Comment by priscilawebdev** [comment]
This PR adds SvelteKit as a platform to the Sentry ~backend~ frontend since the SvelteKit SDK is nearing GA. 😉

File: static/app/components/profiling/ProfilingOnboarding/util.ts
Comment (priscilawebdev) [inline static/app/components/profiling/ProfilingOnboarding/util.ts:51]: have we added docs for the onboarding in our [sentry-docs](https://github.com/getsentry/sentry-docs) repo as well? otherwise this will be broken

File: static/app/data/platforms.tsx
Comment (priscilawebdev) [inline static/app/data/platforms.tsx:28]: this is great!..but... I don't think we have docs for sveltekit in our wizard folder https://github.com/getsentry/sentry-docs/tree/master/src/wizard/javascript

**Review by priscilawebdev — CHANGES_REQUESTED** [review]
please see my comments

File: static/app/components/profiling/ProfilingOnboarding/util.ts
Comment (Lms24) [reply static/app/components/profiling/ProfilingOnboarding/util.ts:51]: I wasn't sure if this would cause breakage or just not work for now. But regardless, I think this should be okay once https://github.com/getsentry/sentry-docs/pull/6867 is merged

File: static/app/data/platforms.tsx
Comment (Lms24) [reply static/app/data/platforms.tsx:28]: We'll soon have them https://github.com/getsentry/sentry-docs/pull/6867 :)

I'll merge this PR only after the docs PR was merged to make sure this isn't causing problems.

Comment (Lms24) [reply static/app/data/platforms.tsx:28]: https://github.com/getsentry/sentry-docs/pull/6867 is merged, I think this should be good to go now

**Review by priscilawebdev — APPROVED** [review]
let's just wait for the docs to be released and then we can merge and deploy this one. We can keep an eye on https://docs.sentry.io/_platforms/_index.json

**Comment by Lms24** [comment]
Good point. Also, I'm still waiting on the backend PR to deploy.

**Comment by Lms24** [comment]
Alright, backend is deployed and and docs are also released and deployed. Gonna merge this in a second.

**Pull request merged**
